_data/projects/brackets-php-beautify.yml

```
@@ -0,0 +1,12 @@
+name: Brackets PHP Beautify
+desc: Simple regex based PHP code formatter (beautifier) for Adobe Brackets. No PHP server required
+site: https://github.com/OSS-Enthusiasts/brackets-php-beautify
+tags:
+  - php
+  - regex
+  - javascript
+  - brackets
+  - requirejs
+upforgrabs:
+  name: up-for-grabs
+  link: https://github.com/OSS-Enthusiasts/brackets-php-beautify/issues?q=is%3Aissue+is%3Aopen+label%3Aup-for-grabs
```

This can be updated to a more succinct value:

```suggestion
  link: https://github.com/OSS-Enthusiasts/brackets-php-beautify/labels/up-for-grabs
```

---

Indeed! Thank you 🙏 ☺